### ISSUE TITLE
Needs to be on Xbox

### ISSUE THREAD

**Issue by benwtrent** [DoubloonDevs/ClammyHands#1]
Dis gaym SUX!!!! I got rekt too quick bc frames dropped. PC SUX!!!! XBOX4LYFE!

Fix bug of not being on XBOX!

**Comment by DoubloonDevs** [comment]
Good to see the constructive criticism. T-T

**Comment by benwtrent** [comment]
Honestly, this is hilarious and I love it.

**Comment by DoubloonDevs** [comment]
Yeah, thanks man. Have been learning Javascript for a few months and I'm feeling more confident about making games. This is just a small 2 day project to see if a could make a simple yet effective engine using HTML5. Going to be working on some serious stuff soon. :)
